scripts/tests/update-changelog.test.js

```diff
@@ -108,3 +111,19 @@ describe('collateChangelogFiles', () => {
     );
   });
 });
+
+describe('getUpcomingVersion', () => {
+  jest.mock('../../package.json', () => ({
+    version: '1.2.3',
+  }));
+
+  test('patch', () => {
+    expect(getUpcomingVersion('patch')).toEqual('1.2.4');
+  });
+  test('major', () => {
```

Comment:
derp, just noticed this 🤪 
```suggestion
  test('minor', () => {
```